freqtrade/rpc/telegram.py

```diff
@@ -258,23 +259,31 @@ def _profit(bot: Bot, update: Update) -> None:
     """
     trades = Trade.query.order_by(Trade.id).all()
 
-    profit_amounts = []
-    profits = []
+    profit_all_btc = []
```

Comment:
I would write the variable assignment a little bit different in this case to group them, but that is just a personal preference, i.e.:
```
profit_all_btc, profit_all = [], []
```

Reply:
I would argue that multi assignment on single line usually just hurts readability.

Reply:
Fair enough